specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json

```diff
@@ -6739,8 +6739,8 @@
           "type": "integer",
           "format": "int32",
           "minimum": 1,
-          "maximum": 3,
-          "description": "Number of fault domains that the host group can span."
+          "maximum": 5,
```

Comment:
Unless we are confident that this value will never ever go up in the future, please remove the `maximum` attribute.